[FEATURE] Afficher une bannière annonçant l'ouverture de l'import de session en masse sur Pix Certif (PIX-6964).

AndreiaPena opened the pull request:
## :unicorn: Problème
Actuellement la fonctionnalité permettant d'importer en masse des session et candidat sur Pix Certif  n'est pas disponible pour les utilisateurs (sous feature toggle). 
Or nous souhaitons rendre disponible cette fonctionnalité sous peu et leur communiquer l'information. 

## :robot: Proposition
Afficher une bannière indiquant que la nouvelle fonctionnalité est désormais dispo.

## :rainbow: Remarques
Cette bannière ne s'affiche pas pour : 

- Les espaces SCO gérant des élèves (is managing student)
- en domaine .org / en anglais.
- si le FT est à false.

## :100: Pour tester
(Le FT `FT_MASSIVE_SESSION_MANAGEMENT` de la RA est à true)

Se connecter sur Pix Certif : 

1/ - en .fr avec certifsup@example.net
- constater que la bannière s'affiche. Cliquer sur le lien et s'assurer qu'on accède au guide.
- constater que les boutons pour créer des sessions en masse est visible.

2/ - en .fr avec certifsco@example.net
- constater que la bannière ne s'affiche pas. 
- Passer dans le centre AEFE
- constater que la bannière s'affiche.

3/ - en .org ET en français : 
- avec certifsco@example.net, constater que la bannière s'affiche.

4/ - en org ET en anglais : 
- constater que la bannière ne s'affiche pas.

5/ - Passer le FT à false et constater que la bannière ne s'affiche pas.

aceol committed:
:sparkles: certif: add method showMassImportBanner for non sco and fr

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr6471.review.pix.fr)
  * [App (.org)](https://app-pr6471.review.pix.org)
  * [Orga (.fr)](https://orga-pr6471.review.pix.fr)
  * [Orga (.org)](https://orga-pr6471.review.pix.org)
  * [Certif (.fr)](https://certif-pr6471.review.pix.fr)
  * [Certif (.org)](https://certif-pr6471.review.pix.org)
  * [Admin](https://admin-pr6471.review.pix.fr)
  * [API](https://api-pr6471.review.pix.fr/api/)
  * [Pix1D](https://1d-pr6471.review.pix.fr)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr6471/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr6471/environment)

aceol committed:
certif: Replace stub with value

isMassiveSessionManagementEnabled was returning a stub (fn) which is truthy.
If the stub was returning false, the test would have failed as is would still be truthy
